amundsen_application/static/js/components/Feedback/styles.scss

```diff
@@ -50,10 +51,32 @@
   }
 
   .feedback-header {
+    display: flex;
     margin-bottom: 8px;
   }
 
   .btn-group {
     margin: 8px auto 16px;
   }
+
+  button.close-button {
+    height: 18px;
```

Comment:
Regarding a re-useable close button, modals aren't going to have `X` buttons for closing according to our new design concepts. The edit owners and edit tag modals already follow this and the preview data modal will also be updated eventually. I thought of checking with Design if they still want the `X` here as well, but dismissed that idea because this isn't the same kind of modal and already has it's own specifications that sets it apart. 

Regarding `btn-flat-icon`, I initially didn't choose that option because the hover isn't intended to be  `$brand-color-4` here. Assuming we don't want to introduce adhoc design changes, my options were:
1. Implement this with the shared `btn-flat-icon` styles and override the hover
2. Define a custom button in this file.

I chose Option 2 over Option 1, because we previously decided against Option 1 (leveraging `btn-flat-icon` just to override some of its styles) when implementing the icons on the profile page. Thoughts?

Reply:
I'm not opposed to a `$brand-color-4` on hover, but I suppose that is Design's decision.

I still think this is a likely candidate for being a reused button, so I would still prefer to define the `close-button` class in `_buttons-default.scss`.

Reply:
No problem. I will define a close-button class, but don't intend to change the existing design unless it's called for.